packages/xo-web/src/xo-app/xoa/support/index.js

```diff
@@ -46,6 +47,17 @@ const Support = decorate([
           </Col>
         </Row>
       )}
+      <Row className='mb-1'>
+        <Col>
+          <ActionButton
```

Comment:
You can use the `ReportBugButton`.

Reply:
See #4234, this button is for creating a new ticket, it's not related to reporting bug.
and I use `reportOnSupportPanel` to attach the interesting files for support.